### PR TITLE
fix 404 errors by using absolute urls

### PR DESCRIPTION
**Pull request by Maximalfr**

### CODE REVIEW

File: COPYING.md [hunk 1]
```diff
@@ -26,48 +26,48 @@ ProtonMail Bridge includes the following 3rd party software:
...
+* [smtp](https://github.com/emersion/go-smtp)                               | Available under [license](https://github.com/emersion/go-smtp/blob/master/LICENSE)
+* [textwrapper](https://github.com/emersion/go-textwrapper)                 | Available under [license](https://github.com/emersion/go-textwrapper/blob/master/LICENSE)
+* [vcard](https://github.com/emersion/go-vcard)                             | Available under [license](https://github.com/emersion/go-vcard/blob/master/LICENSE)
+* [color](https://github.com/fatih/color)                                   | Available under [license](https://github.com/fatih/color/blob/master/LICENSE)
+* [shlex](https://github.com/flynn-archive/go-shlex)                        | Available under [license](https://github.com/flynn-archive/go-shlex/blob/master/LICENSE)
+* [raven](https://github.com/getsentry/raven-go)                            | Available under [license](https://github.com/getsentry/raven-go/blob/master/LICENSE)
+* [resty](https://github.com/go-resty/resty/v2)                             | Available under [license](https://github.com/go-resty/resty/v2/blob/master/LICENSE)
```
Comment (jonathancross) [inline COPYING.md:52]: This URL doesn't work anymore as the `v2/` sub folder is gone.  Links should probably change to `https://github.com/go-resty/resty/`.  If the specific version **is critical**, we can link to a tag (eg https://github.com/go-resty/resty/releases/tag/v2.0.0), but that seems like overkill.

Comment (Maximalfr) [reply COPYING.md]: Fixed

Comment (jonathancross) [reply COPYING.md]: Same with the https://github.com/go-resty/resty/v2/blob/master/LICENSE link...

Comment (Maximalfr) [reply COPYING.md]: I changed it and I also changed color and shlex license links. Tell me if it's ok

**Comment by jameshoulahan** [comment]
Thanks for the fixes! I opened an internal MR; your commit should appear here Soon™ 🙂

**Comment by jonathancross** [comment]
@jameshoulahan Does this mean ProtonMail is not merging PRs here on GitHub?  Is this GitHub repo just a dump of the internal git repo?

**Comment by jameshoulahan** [comment]
For now, that's correct. Commit authorship is preserved; contributors will be acknowledged. It's just easier on our CI system to have a one-way mirror rather than dealing with bidirectional changes.

**Comment by jonathancross** [comment]
> Commit authorship is preserved; contributors will be acknowledged.

If the commit is PGP signed, will that signature also be preserved?

**Comment by jameshoulahan** [comment]
As much as we try to ensure preservation of authorship, we cannot guarantee preservation of signatures. Rebasing the commits on top of our development branch means the commits can no longer be verified.

**Comment by jameshoulahan** [comment]
Merged internally, will be here soon, closing 🙂

**Pull request closed without merging**